src/store.ts

```diff
@@ -39,41 +39,50 @@ const dbAllAsync = function(db: sqlite3.Database, sql: string) {
   })
 }
 
-export function init() {
-  return new Promise((resolve, reject) => {
-    if (!db) {
-      db = new sqlite3.Database(isTesting ? ':memory:' : dbPath, err => {
-        if (err) {
-          return reject(err)
-        }
-
-        resolve()
+export async function init(resetDB = false) {
```

Comment:
nit: This would be a great place to use an object for clarity/readability as reseting a db is different than simply init'ing, no?

```suggestion
export async function init({ resetDB = false }) {
```